### ISSUE TITLE
Allow contribution submitters to edit some fields of contributions

### ISSUE THREAD

**Issue by OmeGak** [indico/indico#2533]
Feature requested in `CHG34380`.

User's feedback on why increasing modification rights of contribution submitters would be very useful in big conferences.

> I agree that the submitters must not be able to change the scheduling details; however, they must be able to change at least the speaker details. The reason is, many abstracts are submitted without speakers assigned, because some collaborations assign speakers only after a contribution is accepted. While conference managers can add speakers to a dozen of contributions, it is logistically impossible to do so with 500+ contributions (this includes posters). Same is true for adding/changing co-authors and even submitters themselves, but this is less critical (some abstracts are submitted by generic group leaders, not real authors).

> It would be also nice if the submitters can modify title and abstract text. The reason is, some abstracts are merged into one contribution. Obviously, Indico can not do automatic merger of the titles and texts, so the submitters must be authorised to do it. This is not a very frequent use case, but it does happen.

**Comment by ThiefMaster** [comment]
Could be done like "session coordinator rights" where you can grant submitters them additional privileges
